cmds/stage

```diff
@@ -381,5 +384,7 @@ stage_main() {
                 ;;
         esac
     done
+
+    popd >/dev/null
```

Comment:
This makes the return of popd the return of the function.  This is a little surprising combined with checking call_cmd errors (https://github.com/apertussolutions/bordel/pull/47).

Ran `deploy update` - it calls `stage_main repository` which failed to copy an image, but `popd` succeeded and it still created a broken update.tar.  While the error message pointed, the subsequent `&& deploy iso` still ran, so the messages wasn't readily visible.  i.e. I thought everything worked.